### PR TITLE
Getlog

### PR DESCRIPTION
**Pull request by maaikez**
Implementation of GetLog, based on 1.6 implementation.

### CODE REVIEW

File: lib/ocpp/v201/charge_point.cpp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
 
+#include "ocpp/v201/messages/LogStatusNotification.hpp"
```
Comment (Pietfried) [inline lib/ocpp/v201/charge_point.cpp:4]: we can include with <>

File: lib/ocpp/v201/charge_point.cpp [hunk 2]
```diff
@@ -212,6 +213,16 @@ void ChargePoint::on_event(const std::vector<EventData>& events) {
...
 }
 
+void ChargePoint::on_log_status_notification(UploadLogStatusEnum status, int32_t requestId)
+{
+    LogStatusNotificationRequest request;
+    request.status = status;
+    request.requestId = requestId;
+
+    ocpp::Call<LogStatusNotificationRequest> call(request, this->message_queue->createMessageId());
+    this->send_async<LogStatusNotificationRequest>(call);
```
Comment (Pietfried) [inline lib/ocpp/v201/charge_point.cpp:223]: this can and should be
 `this->send<LogStatusNotificationRequest>(call);`

**Review by Pietfried — CHANGES_REQUESTED** [review]
Looks pretty good, only some minor changes requested.

**Comment by maaikez** [comment]
> Looks pretty good, only some minor changes requested.

Done!

**Review by Pietfried — APPROVED** [review]

**Pull request merged**